packages/cli-repl/src/logger.ts

```diff
@@ -12,7 +14,7 @@ function logger(bus: any, logDir: string) {
   bus.on('*', function() {});
 
   bus.on('connect', function(info) {
-    const params = { sessionID, info };
+    const params = { sessionID, info: redactPwd(info) };
```

Comment:
Why not `redactPassword`, may be just me being exceptionally confused by abbreviations though.